CHANGELOG.md

```diff
@@ -23,12 +24,14 @@ IMPROVEMENTS:
 * `resource/vcd_vapp` supports guest properties [#319]
 * `resource/vcd_vapp_vm` supports guest properties [#319]
 * Upgrade Terraform SDK dependency to 0.12.6 [#302]
+* `vcd_org_vdc` Add import capability - ([#323])
```

Comment:
We need to agree on using or not `()` for enclosing GitHub issue IDs. Half of the items have it, have don't.

Reply:
Fixed

Reply:
My vote for no "()"